includes/features/analytics/class-wc-admin-analytics.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * WooCommerce Analytics.
+ * NOTE: DO NOT edit this file in WooCommerce core, this is generated from woocommerce-admin.
+ *
+ * @package Woocommerce Admin
+ */
+
+/**
+ * Contains backend logic for the onboarding profile and checklist feature.
+ */
+class WC_Admin_Analytics {
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Admin_Analytics instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		add_filter( 'woocommerce_component_settings_preload_endpoints', array( $this, 'add_preload_endpoints' ) );
+	}
+
+	/**
+	 * Preload data from the performance indiciators endpoint.
+	 *
+	 * @param array $endpoints Array of preloaded endpoints.
+	 * @return array
+	 */
+	public function add_preload_endpoints( $endpoints ) {
+		$endpoints['countries'] = WC_ADMIN_API_NAMESPACE . '/data/countries';
```

Comment:
This getting replaced can actually throw a concat linter error in core -- so I might need to do this differently.

```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
44 | ERROR | String concat is not required here; use a single string
   |       | instead
   |       | (Generic.Strings.UnnecessaryStringConcat.Found)
----------------------------------------------------------------------
```